### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by nickvigilante**
Update Contributing guide to reflect the new monorepo structure. Also fix some outdated content about our docs tooling.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -98,6 +100,7 @@ Once you're ready to contribute:
 6. Use Jekyll to [build a version of the site locally](#build-and-test-the-docs-locally) so you can view your changes in a browser:
 
     ```
+    cd src/<docs version to build>
```
Comment (mdlinville) [inline CONTRIBUTING.md:103]: Today, I think the only thing you can build is `src/current`, right?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -130,7 +133,7 @@ If you want to regularly contribute to the CockroachDB docs, there are a few thi
 
 Once you've installed Jekyll and have a local clone of the docs repository, you can build and test the docs as follows:
 
-1. From the root directory of your clone, :
+1. From one of the subdirectories of the `src` directory of your clone:
```
Comment (mdlinville) [inline CONTRIBUTING.md:136]: I tried this from `src/archive` and it did not work.

Comment (nickvigilante) [reply CONTRIBUTING.md:136]: I'll add that as an exception.

File: CONTRIBUTING.md (hunk 1 above)
Comment (nickvigilante) [reply CONTRIBUTING.md:103]: Today, you can build in `current`, `api`, and `interactive-tutorials`.

**Review by mdlinville — APPROVED** [review]

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -245,15 +244,15 @@ For information about how we use images in our docs, see [Images](https://github
 
 #### Feedback widget
 
-We show "Yes/No" feedback buttons at the bottom of every page by default. To remove these buttons from a page, set `feedback: false` in the page's front-matter.
+We show thumbs-up and thumbs-down feedback buttons after a page's TOC on the bottom-right portion of the docs window for every page by default. To remove these buttons from a page, set `feedback: false` in the page's front matter.
```
Comment (mdlinville) [inline CONTRIBUTING.md:247]:
```suggestion
We show thumbs-up and thumbs-down feedback buttons after a page's TOC on the bottom-right portion every page by default. To remove these buttons from a page, set `feedback: false` in the page's front matter.
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -191,26 +198,26 @@ Field | Description | Default
...
 The CockroachDB Jekyll theme can auto-generate a page-level table of contents listing all h2 and h3 headers or just all h2 headers on the page. Related files: `js/toc.js` and `_includes/toc.html`.
 
-- To add a page TOC, set `toc: true` in the page's front-matter.
+- To add a page TOC, set `toc: true` in the page's front matter.
 
-- To omit a page TOC from the page, set `toc: false` in the page's front-matter.
+- To omit a page TOC from the page, set `toc: false` in the page's front matter.
 
-- By default, a page TOC includes h2 and h3 headers. To limit a page TOC to h2 headers only, set `toc_not_nested: true` in the page's front-matter.
+- By default, a page TOC includes h2 and h3 headers. To limit a page TOC to h2 headers only, set `toc_not_nested: true` in the page's front matter.
```
[the remark this answers is not in the document]
Comment (nickvigilante) [reply CONTRIBUTING.md:205]: Checking.

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -95,17 +99,20 @@ Once you're ready to contribute:
...
 
-We'll review your changes, providing feedback and guidance as necessary. Also, Teamcity, the system we use to automate tests, will run the markdown files through Jekyll and then run [htmltest](https://github.com/cockroachdb/htmltest) against the resulting HTML output to check for errors. Teamcity will also attempt to sync the HTML to an AWS server, but since you'll be working on your own fork, this part of the process will fail; do not worry about the Teamcity fail status.
+We'll review your changes, providing feedback and guidance as necessary. Also, Netlify, the system we use to automate tests, will run the markdown files through Jekyll and then run a series of automated tests, including [htmltest](https://github.com/cockroachdb/htmltest), against the resulting HTML output to check for errors. If all automated tests pass, Netlify will attempt to provide a deploy preview link in a comment in the pull request.
```
[the remark this answers is not in the document]
Comment (nickvigilante) [reply CONTRIBUTING.md:115]: Done!

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -95,17 +99,20 @@ Once you're ready to contribute:
...
+    cd src/<docs area to build>
     make cockroachdb
     ```
 
+    **Note:** The `archived` folder is not configured to use Jekyll or any `make` commands.
+
 7. [Push your local branch to your remote fork](https://help.github.com/articles/pushing-to-a-remote/).
 
-8. Back in your fork of the CockroachDB docs repo in the GitHub UI, [open a pull request](https://github.com/cockroachdb/docs/pulls) and assign it to `jseldess`. If you check the `Allow edits from maintainers` option when creating your pull request, we'll be able to make minor edits or fixes directly, if it seems easier than commenting and asking you to make those revisions, which can streamline the review process.
+8. Back in your fork of the CockroachDB docs repo in the GitHub UI, [open a pull request](https://github.com/cockroachdb/docs/pulls) and assign it to `cockroachdb/docs-prs`. If you check the `Allow edits from maintainers` option when creating your pull request, we'll be able to make minor edits or fixes directly, if it seems easier than commenting and asking you to make those revisions, which can streamline the review process.
```
[the remark this answers is not in the document]
Comment (nickvigilante) [reply CONTRIBUTING.md:113]: I just linked to the GitHub docs on how to open a PR instead.

File: CONTRIBUTING.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (nickvigilante) [reply CONTRIBUTING.md:205]: It's not a front matter item. It's just `toc` and `toc_not_nested`. The only place where `toc_levels` is referenced is in the Jekyll source code on rendering the TOC.

**Pull request merged**